build/media/libraries/redcore/media/redcore/css/component.bs3.css

```diff
@@ -9763,7 +9723,7 @@
 .redcore .chart-box {
   padding: 0;
 }
-.redcore .modal-body {
+.redcore .redcore .modal-body {
```

Comment:
why 2x .redcore?

Reply:
Yes.

Reply:
@Kixo Hi.

Reply:
this file is compiled from `component.bs3.less`?

Reply:
@jpomer are you compiling component.bs3.less or are you adding that lines yourself? If you are compiling then you didnt add changes to the component.bs3.less file in your PR

Reply:
I'm adding changes to component.bs3.less and gulp is compiling.

Reply:
Sure.